example/signal-halogen/README.md

```diff
@@ -0,0 +1,11 @@
+## Build Instructions
+
+You can build this from the root directory
+
+```bash
+> npm i
+> npm run build
+> npm run example-signal-halogen
```

Comment:
Add `example-signal-halogen` into `scripts` of [`package.json`](https://github.com/f-o-a-m/purescript-react-map-gl/blob/master/package.json)